opentracing-spring-web-itest/mvc/src/test/java/io/opentracing/contrib/spring/web/interceptor/itest/mvc/MVCJettyITest.java

```diff
@@ -68,129 +55,4 @@ protected TestRestTemplate getRestTemplate() {
         return testRestTemplate;
     }
 
-    @Test
-    public void testFilterException() throws Exception {
```

Comment:
These tests are exact copies of the tests they override in the `AbstractBaseITests.java`, except that instead of handling two spans they were just handling one - the second being this additional `Span` that gets created when the interceptor is called outside the scope of the server span.

As mentioned in the description for this PR, I did not see the benefit of representing this span, and without it, the spring boot and mvc tests were exactly the same.

Can you explain why this extract Span is of value, when it occurs outside the scope of the server span? I can add the mechanism to capture this additional Span back in, but wanted to be sure it is worth it.